content/CORE/GettingStarted/Applications.md

```diff
@@ -140,7 +140,7 @@ Installing a basic PlexMedia Plugin:
 
    ![JailsStart](/images/CORE/Jails/JailsStart.png "Starting a Jail")
 
-2. When the jail **State** changes to **up**, click *> SHELL* to see the jail command line.
+2. When the jail **State** changes to **up**, click **> SHELL** to see the jail command line.
```

Comment:
same point about the > preceding SHELL. I don't see the > on the UI screen.

Reply:
again this is saying click [button], not click [UI screen] so I think the thing we say to click should match the thing they click, no?

Reply:
we need to remove the > because it is not part of the UI for this option.

Reply:
```suggestion
2. When the jail **State** changes to **up**, click **SHELL** to see the jail command line.
```